docs/source/guides/configuration.rst

```diff
@@ -36,19 +42,28 @@ Deployment-specific parameters
   transactions are sent here.
 - ``internal_port`` sets the port for internal communications: ordering
   service, consensus and block loader.
-- ``pg_opt`` is used for setting credentials of PostgreSQL: hostname, port,
-  username, password and database name.
+- ``database`` (optional) is used to set the database configuration (see below)
+- ``pg_opt`` is a deprecated way of setting credentials of PostgreSQL:
+  hostname, port, username, password and database name.
   All data except the database name are mandatory.
   If database name is not provided, a default one gets used.
-- ``working database`` is the name of working database - the one that will be used to store the world state view and optionally blocks.
-    .. warning:: Deprecated behaviour: is overrided by ``dbname`` parameter in ``pg_opt`` value.
-    .. warning:: Deprecated: optional.
-- ``maintenance database`` is the name of databse that will not be used to store any data, but only to maintain the working database.
-   .. warning:: Deprecated: optional.
 - ``log`` is an optional parameter controlling log output verbosity and format
   (see below).
 
-.. warning:: Unspecified database name support is deprecated and will be removed!
+.. warning::
+   Configuration field ``pg_opt`` is deprecated, please use ``database`` section!
+
+   The ``database`` section overrides ``pg_opt`` when both are provided in configuration.
```

Comment:
This is much better now, many thanks!

Reply:
Can we omit at all `pg_opt` parameter now?
If so, please leave a comment about that here too. 
Otherwise, the only configuration example that contains both parameters at the same time looks a bit confusing.

Reply:
made `pg_opt` an optional configuration member (b01df0a) and updated the docs (b01dc6a3b)